packages/application-generic/src/usecases/create-notification-jobs/create-notification-jobs.usecase.ts

```diff
@@ -87,9 +87,10 @@ export class CreateNotificationJobs {
       if (!step.template)
         throw new PlatformException('Step template was not found');
 
-      const providerId = command.templateProviderIds.get(
-        STEP_TYPE_TO_CHANNEL_TYPE.get(step.template.type) as ChannelTypeEnum
-      );
+      const channel = STEP_TYPE_TO_CHANNEL_TYPE.get(
+        step.template.type
+      ) as ChannelTypeEnum;
```

Comment:
I feel we should change the `STEP_TYPE_TO_CHANNEL_TYPE` to object as we now need to cast the value always

Reply:
Looks like the cast was not needed at all so i removed it.